src/css/objects/_media.scss

```diff
@@ -170,7 +169,6 @@
     &--small {
       font-size: ms(-2);
     }
-
   }
 
   &__overline {
```

Comment:
Per @tinykb's new media object designs, overlines should always be blue. Use $color-secondary in place of $color-primary. Also, overline font size needs to be bumped up to `ms(0)`